sofiadinizms/NFT-onepage
# Footer, Banner, bugfix

## What I did
1. Fixed the GenericSections map bug
2. Finished Section details (not in the best way, this component definitely needs to be refactored)
3. Created the Footer section
4. Created the Banner

## How to test it
1. Install de project dependencies (just to make sure, I recommend using both yarn and npm install) on both the client and server folders
2. Run the front end with `npm start` or `yarn start` 
3. Run the backend with `npm start`
4. Try to change the Sections content on `localhost:3001/admin` and check if the front reflects those changess
5. Compare all the sections with the Figma mockup

## Technical debts
1. The Section component is a little dirty and deserves to be refactored in the future
2. Footer is yet to be connected to the backend
